src/fmu/dataio/tools/rms/volumetrics.py

```diff
@@ -0,0 +1,228 @@
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from typing import Final
+
+import pandas as pd
+from packaging.version import parse as versionparse
+
+import fmu.dataio as dio
+from fmu.dataio._logging import null_logger
+
+try:
+    import rmsapi
+    from _rmsapi import Project as RmsProject
+except ImportError:
+    pass
+
+try:
+    import roxar as rmsapi
+    from _roxar import Project as RmsProject
+except ImportError:
+    pass
+
+_logger: Final = null_logger(__name__)
+
+# rename columns to FMU standard
+RENAME: Final = {
+    "Proj. real.": "REAL",
+    "Zone": "ZONE",
+    "Segment": "REGION",
+    "Boundary": "LICENSE",
+    "Facies": "FACIES",
+    "BulkOil": "BULK_OIL",
+    "NetOil": "NET_OIL",
+    "PoreOil": "PORV_OIL",
+    "HCPVOil": "HCPV_OIL",
+    "STOIIP": "STOIIP_OIL",
+    "AssociatedGas": "ASSOCIATEDGAS_OIL",
+    "BulkGas": "BULK_GAS",
+    "NetGas": "NET_GAS",
+    "PoreGas": "PORV_GAS",
+    "HCPVGas": "HCPV_GAS",
+    "GIIP": "GIIP_GAS",
+    "AssociatedLiquid": "ASSOCIATEDOIL_GAS",
+    "Bulk": "BULK_TOTAL",
+    "Net": "NET_TOTAL",
+    "Pore": "PORV_TOTAL",
+}
+
+
+@dataclass
+class _RmsVolumetricsExport:
+    """A simplified interface when exporting volume tables (etc) from RMS."""
+
+    project: RmsProject
+    gridname: str
+    volumejob_name: str
+
+    # optional and defaulted
+    config: dict = field(default_factory=dict)  # currently required!
+    forcefolder: str = ""  # unsure if we shall allow this?
```

Comment:
Different topic, but it would be good to get away from these things, and be more brutal on the standard of relative_filepath. I think much logic will have to be placed on this path going forward, but at this point it is impossible.

Reply:
I keep forcefolder, under great doubt, but is is used on Drogon...

Reply:
Issue warning when used? "May not be supported in the future"? What is the reason this has be used in Drogon? If it _has_ to be used, for whatever reason, this might be important feedback to the standard?

Reply:
May change my mind and drop; but possibly support `subfolder` since that is not deprecated?